Mathlib/Algebra/Order/Ring/Abs.lean

```diff
@@ -94,16 +94,20 @@ lemma abs_le_one_iff_mul_self_le_one : |a| ≤ 1 ↔ a * a ≤ 1 := by
 
 lemma abs_sq (x : α) : |x ^ 2| = x ^ 2 := by simpa only [sq] using abs_mul_self x
 
+lemma sq_lt_sq_iff_of_nonneg (ha : 0 ≤ a) (hb : 0 ≤ b) : a ^ 2 < b ^ 2 ↔ a < b :=
+  (pow_left_strictMonoOn₀ two_ne_zero).lt_iff_lt ha hb
```

Comment:
How about [`pow_lt_pow_iff_left₀`](https://github.com/leanprover-community/mathlib4/blob/197eebfa5455aa09b7a4c2ad3c3eb9123245df90/Mathlib/Algebra/Order/GroupWithZero/Unbundled.lean#L1220-L1221) instead?
```lean
lemma sq_lt_sq_iff_of_nonneg (ha : 0 ≤ a) (hb : 0 ≤ b) : a ^ 2 < b ^ 2 ↔ a < b :=
  pow_lt_pow_iff_left₀ ha hb two_ne_zero
```

Would it make more sense to put this new lemma (which does not directly pertain to `abs`) in `Algebra.Order.GroupWithZero.Unbundled`, immediately after `pow_lt_pow_iff_left₀`?